### DISCUSSION TITLE
Searching large data sets (top big for memory)

### DISCUSSION THREAD

**Discussion by DrBamba** [milvus-io/milvus#17025] [Ideas & Feature requests]
Say you have a huge collection, and it takes more storage than the memory your query nodes have, how can you Search it?
I guess you could divide it into partitions and load each one of them separately but a) it will add a lot of overhead and b) it will be exhausting to implement it from client side

Are there plans to dynamically load parts of the collection? Or to deal with it in any other way? Or even search on disk somehow?

**Comment by yhmo** [comment]
According to this plan: https://wiki.lfaidata.foundation/display/MIL/Milvus+2.X+Roadmap+and+Time+schedule
The "Segment LRU load" is in v2.2.
